src/test/java/lotto/domain/LottoGameTest.java

```diff
@@ -11,7 +11,8 @@ class LottoGameTest {
     @DisplayName("구입 금액에서 로또 개수 계산 테스트")
     @Test
     void available_lotto_quantity() {
-        assertThat(new LottoGame(14560)).extracting("count").isEqualTo(14);
+        assertThat(new LottoGame(14560).getLottos().size())
+                .isEqualTo(14);
```

Comment:
```suggestion
        List<Lotto> lottos = new LottoGame(14560).getLottos();
        assertThat(lottos).hasSize(14);
```
한번에 표현하는 것도 좋지만 때로는 변수로 뽑아두는 것이 더 가독성이 높아질 수 있습니다 😄